src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,47 @@
+package Pages;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+
+
+
+public class HomePage extends BasePage {
+
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+
+    }
+        //Locators
+        By selectPlaylist = By.xpath("//a[@href='#!/playlist/78934']");
+
+        By clickEdit = By.xpath("//li[@data-testid='playlist-context-menu-edit-78934']");
+
+        By playlistNameField = By.xpath("//input[@name='name']");
+
+        By displayedRenameSuccessMsg = By.cssSelector("div.success.show");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```